docs/reference-manual/src/main/asciidoc/copy-config.adoc

```diff
@@ -79,8 +79,10 @@ asadmin-options::
     requires superuser privileges.
   `JAVA_DEBUGGER_PORT`;;
     This property specifies the port number of the port that is used for
-    connections to the Java Platform Debugger Architecture (JPDA)
-    (https://docs.oracle.com/en/java/javase/17/docs/specs/jpda/jpda.html)
+    connections to the Specifies whether the domain is restarted with
+  https://docs.oracle.com/en/java/javase/17/docs/specs/jpda/jpda.html[Java Platform Debugger Architecture (JPDA)]
+  debugging enabled.
+
```

Comment:
Seems that https://glassfish.org/docs/SNAPSHOT/reference-manual.html#copy-config is not updated with that (yet?) - so can't check how this is rendered - but I don't understand that part in **bold** already:

> This property specifies the port number of the port that is used for
> **connections to the Specifies whether** the domain is restarted with

For me, there is something not right with that sentence...